spec/plugins/development_mode_plugin_spec.rb

```diff
@@ -39,6 +40,7 @@ def render(name, stack_trace)
           <li>#{stack_trace.join("</li><li>")}</li>
         </ul>
       </div>
+      #{component}
```

Comment:
I guess i'm not understanding why this line needs to be in this file, if it's not adding any content? What test would fail if this line was removed?